### PR TITLE
Upgrade to support Windows

### PR DESCRIPTION
**Pull request by wfspotz**

### CODE REVIEW

**Comment by isuruf** [comment]
Did you rerender in `wfspotz/master` or in `conda-forge/master`?

**Comment by isuruf** [comment]
Can you try rerendering in this branch? If that doesn't work, I'll rerender and send a PR for you.

**Comment by wfspotz** [comment]
I'm confused. What is the order of steps I should be taking? None of the set of steps I try lead to a rerendered PR.

**Comment by isuruf** [comment]
I found out why there's no change when conda-smithy is run. There's no `numpy 1.7` for windows. Can you change that to `1.9` and do `conda smithy rerender`?

**Comment by wfspotz** [comment]
Should I do this on a Windows machine, like appveyor?

**Comment by isuruf** [comment]
No need. You can run `conda smithy rerender` on any platform.

**Comment by wfspotz** [comment]
$ pwd
    /Users/wfspotz/Development/pytrilinos-feedstock
    $ grep numpy recipe/meta.yaml 
        - numpy 1.9.*
        - numpy >=1.9
    $ conda smithy rerender -c
    Fetching package metadata ...........
    Fetching package metadata ...........
    Fetching package metadata .............
    Fetching package metadata .............
    
    Re-rendered with conda-smithy 2.4.0.
    
    No changes made. This feedstock is up-to-date.

**Comment by isuruf** [comment]
I sent a PR to your fork, https://github.com/wfspotz/pytrilinos-feedstock/pull/1

**Comment by isuruf** [comment]
Looks like openmpi is missing in windows.

**Comment by wfspotz** [comment]
OK. For now, I think I will make the Windows version serial-only.

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,47 @@
+mkdir build
+cd build
+
+cmake ^
+  -G "%CMAKE_GENERATOR%" ^
+  -D CMAKE_BUILD_TYPE:STRING=RELEASE ^
+  -D CMAKE_INSTALL_PREFIX:PATH=%PREFIX% ^
```
Comment (isuruf) [inline recipe/bld.bat]: Use `conda-forge`'s convention of `%LIBRARY_PREFIX%` instead of just `%PREFIX%`.

**Comment by wfspotz** [comment]
Due to openmpi not being supported for Windows, I am going to close this pull request for now.

**Pull request closed without merging**